util/download_data.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const _ = require('lodash');
+const async = require('async');
+const child_process = require('child_process');
+const fs = require('fs-extra');
+const config = require( 'pelias-config' ).generate(require('../schema'));
+const logger = require('pelias-logger').get('download');
+
+if (require.main === module) {
+  download((err) => {
+    if (err) {
+      logger.error('Failed to download data', err);
+      process.exit(1);
+    }
+    logger.info('All done!');
+  });
+}
+
+function download(callback) {
+  let sources;
+
+  if (!_.isEmpty(config.imports.openstreetmap.download)) {
+    sources = _.map(config.imports.openstreetmap.download, (source) => source.sourceURL);
+  }
+  else {
+    sources = [
+      'http://planet.openstreetmap.org/pbf/planet-latest.osm.pbf'
```

Comment:
So on the one hand, the planet file makes sense as a default because it's what we would use in production, and because it has everything. But it's also really really really big. I could imagine someone just going with the defaults and being very confused when their local machine crumbles under the weight of trying to import all of OSM.

I'm not sure there's a better solution though. We could link to one of our smaller "standard" extracts like Berlin, London, NYC, etc, but then we are defaulting to data that isn't relevant to everyone.

There might not be an ideal solution, anyone have any ideas?